src/MsgPack/Serialization/ReflectionExtensions.cs

```diff
@@ -199,6 +199,13 @@ public static CollectionTraits GetCollectionTraits( this Type source, Collection
 					);
 			}
 
+			// If the type is an interface then a concrete collection has to be
+			// made for it if the interface is a collection type
+			if ( source.IsInterface || forceCollection )
```

Comment:
Use `GetIsInterface()` internal extension method instead of `IsInterface` because `Type` of WinRT/UWP/.NET Core does not declare `IsInterface` property.